.template-lintrc.js

```diff
@@ -5,5 +5,12 @@ module.exports = {
   rules: {
     "no-bare-strings": true,
     "no-implicit-this": { allow: ["housing-stat-link", "current-quarter"] },
```

Comment:
Could you check why this is allowed? :thinking:

Reply:
The linter fails for the housing-stat-link helper:
```
addon/components/housing-stat-link.hbs
  3:9  error  Ambiguous path 'housing-stat-link' is not allowed. Use '@housing-stat-link' if it is a name
d argument or 'this.housing-stat-link' if it is a property on 'this'. If it is a helper or component that
 has no arguments, you must either convert it to an angle bracket invocation or manually add it to the 'n
o-implicit-this' rule configuration, e.g. 'no-implicit-this': { allow: ['housing-stat-link'] }.  no-implicit-this
```
```hbs
<a
  class="uk-button uk-button-default"
  href={{housing-stat-link}}
  target="_blank"
  rel="noopener noreferrer"
>
  {{t "ember-gwr.nav.housingStatLink"}}
</a>
```

Reply:
Ah damn.